### ISSUE TITLE
[PR] New Puerto Rico testing count report for July 24

### ISSUE THREAD

**Issue by sacundim** [COVID19Tracking/issues#669]
**State or US**: Puerto Rico

The Puerto Rico Department of Health shared a new Bioportal report with its testing counts as of July 23. As usual, it just shares them with some journalists who share it on Twitter:

* https://twitter.com/angelicaserran0/status/1286690476968599555
* https://twitter.com/RobbyCortes/status/1286697081705566208

As usual as well, I have a CSV with all the data of all these irregular reports:

* https://github.com/sacundim/covid-19-puerto-rico/blob/master/assets/data/cases/PuertoRico-bioportal.csv

---
Images of the report (2 pages):

![2020-07-23_bioportal](https://user-images.githubusercontent.com/4076056/88423807-40465000-cda1-11ea-8d10-e5d17d7f1a34.jpg)

![2020-07-23_bioportal_p2](https://user-images.githubusercontent.com/4076056/88423816-450b0400-cda1-11ea-939c-fd4557fc7e34.jpg)

**Comment by muamichali** [comment]
Thank you @sacundim and @MattHilliard so much!

**Comment by MattHilliard** [comment]
OK, so there was a miscommunication after the last update I made and the daily data entry team continued carrying over the previous (6/29) numbers on 7/12 and after. So I not only put in @sacundim's new numbers for 7/23 and 7/24, I also fixed the data for previous two weeks. This time I will make sure we handle this correctly in tomorrow's data entry shift.

I did this right after today's publication, so these numbers aren't factored into the test numbers in today's tweets. The website however should pick them up in an hour or so.

PCR columns before:
![image](https://user-images.githubusercontent.com/404503/88442176-84594500-cde1-11ea-9f45-9a867b43063a.png)

PCR columns after:
![image](https://user-images.githubusercontent.com/404503/88442234-ca160d80-cde1-11ea-9107-7c8a633f8d8f.png)

Negative column before:
![image](https://user-images.githubusercontent.com/404503/88442334-1a8d6b00-cde2-11ea-909b-b119ebc5d2e9.png)

Negative column after:
![image](https://user-images.githubusercontent.com/404503/88442349-2da03b00-cde2-11ea-80e5-14a026057930.png)

One anomaly here is that on 7/15 the data entry team reported a new number for Positive Cases (PCR) that is lower than the data point we had for 7/9. I'm not sure what happened there yet but wanted to get the data fixed first.

As always, thanks so much @sacundim for keeping us informed of these data releases!
